src/Abp.AspNetCore/AspNetCore/EntityHistory/HttpRequestEntityChangeSetReasonProvider.cs

```diff
@@ -22,9 +22,6 @@ public override string Reason
                     return OverridedValue.Reason;
                 }
 
-                // TODO: Use back HttpContextAccessor.HttpContext?.Request.GetDisplayUrl()
-                // after moved to net core 3.0
-                // see https://github.com/aspnet/AspNetCore/issues/2718#issuecomment-482347489
```

Comment:
Can we modify the comment to mention that this workaround is needed for asp net core 2.2 or below?